apps/prairielearn/src/lib/file-store.js

```diff
@@ -112,8 +113,8 @@ module.exports.delete = async (file_id, authn_user_id) => {
 /**
  * Option of returning a stream instead of a file
  *
- * @param {string} file_id - The file to get.
- * @return {stream} - Requested file stream.
+ * @param {number} file_id - The file to get.
```

Comment:
Nit: we typically like to accept at *least* strings for IDs since that's what IDs will be when they're queried via a Zod schema.

```suggestion
 * @param {number | string} file_id - The file to get.
```